### PR TITLE
Link to Github repository, Setup linters, Setup continuous integration, Automating task

### PR DESCRIPTION
**Pull request by thebere**
#### What does this PR do?
1. Link to Github repository
2. Setup Eslint
3. Setup HoundCI
4. Setup Gulp

#### Description of Task to be completed?
1. Setting up Github repository
2. Setting up linters
3. Automating task

#### How should this be manually tested?

[Github repository link](https://github.com/Andela-JUdensi/inverted-index)

###### Follow url for screencast (automating task)
[Screencast showing gulp watch and livereload using browser-sync 1 ](https://www.youtube.com/watch?v=8RpK1UMfzE4)
[Screencast showing gulp watch and livereload using browser-sync 2 ](https://www.youtube.com/watch?v=pVn080h3Y0o)

###### Eslint error
eslint error

###### Eslint error
eslint error 2

###### Hound CI error
hound comment

### CODE REVIEW

**Review by q-ode — COMMENTED** [review]
Please rename your PR properly.

The outputs you're submitting should be in the title. Not "PR to merge..." that's what the PR already does.

File: 06-building-a-program-start-to-finish/automating-tasks/output/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### Output
+- **Using**: Gulp
+- **Deliver**: a screencast showing gulp watching and refreshing your browser when changes are made to your HTML, CSS & JS files
+- **With**: *independently*
+
+#### Video links
+
+[Screencast showing gulp watch and livereload using browser-sync 1 ](https://www.youtube.com/watch?v=8RpK1UMfzE4)
+
+[Screencast showing gulp watch and livereload using browser-sync 2 ](https://www.youtube.com/watch?v=pVn080h3Y0o)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/automating-tasks/output/README.md:10]: Looks good!

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- **Using**: Terminal & GitHub
+- **Deliver**: two screenshots of ESLint [Airbnb](https://github.com/airbnb/javascript)  style violations in the terminal and [HoundCI](https://houndci.com/) comment on a PR
+- **With**: *independently*
+
+#### Screenshots of Eslint Airbnb style violations and HoundCI comments
+![ESlint error](https://github.com/andela/test-simulations-mgmt-repo/blob/bayo-kakashi-judensi/06-building-a-program-start-to-finish/setting-up-linters/output/eslint%20error%202.png)
+
+![Eslint Error](https://github.com/andela/test-simulations-mgmt-repo/blob/bayo-kakashi-judensi/06-building-a-program-start-to-finish/setting-up-linters/output/eslint%20error.png)
+
+![Hound CI comment](https://github.com/andela/test-simulations-mgmt-repo/blob/bayo-kakashi-judensi/06-building-a-program-start-to-finish/setting-up-linters/output/hound%20comment.png)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md:11]: I see the integration from HoundCI and the other from you IDE, it's not from terminal, kindly address.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+#### Link to Github repository
+[Github repository link](https://github.com/Andela-JUdensi/inverted-index)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:7]: Did you get the ReadMe guidelines I sent out? This doesn't conform.

**Review by q-ode — COMMENTED** [review]
Kindly address comments.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md (hunk 3 above)
Comment (thebere) [reply 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:7]: I have not seen it, and I have not actually worked on the readme.

**Pull request closed without merging**